docs/sqlserver.md

```diff
@@ -0,0 +1,55 @@
+# `sqlserver` Metrics Receiver
+
+The `sqlserver` metrics receiver can fetch stats from a SQL Server via the [Windows Performance Counters](https://docs.microsoft.com/en-us/windows/win32/perfctrs/performance-counters-portal).
+
+## Prerequisites
+Because this `sqlserver` receiver makes use of the windows perforamnce counters, it is required for it to run locally on the same machine with the SQL Server installation.
```

Comment:
`it is required for it` is a bit jarring. How about just `it is required to run`?

Reply:
changed